Gemfile.lock

```diff
@@ -419,8 +422,8 @@ GEM
       netrc (~> 0.7)
     retriable (1.4.1)
     retries (0.0.5)
-    roar (1.0.3)
-      representable (>= 2.0.1, <= 3.0.0)
+    roar (1.0.4)
+      representable (>= 2.0.1, < 2.4.0)
```

Comment:
Yep. I took a look at this too.. They're doing a major refactor in `3.0`, and `2.4` introduces some deprecation warnings that are internal to `roar`. They're being good citizens by not inflicting on us deprecation warnings that we're helpless to do anything about.